src/services/webdav/backend.rs

```diff
@@ -256,7 +257,7 @@ impl Debug for WebdavBackend {
 impl Accessor for WebdavBackend {
     type Reader = IncomingAsyncBody;
     type BlockingReader = ();
-    type Pager = ();
+    type Pager = DirStream;
```

Comment:
Please add `AccessorCapability::List` in `set_capabilities` to declare that this backend supports list.